README.md

```diff
@@ -50,8 +50,21 @@ graphite.send(name, res, timestamp)
 ```
 
 ## Usage examples
-`graphite.add_sec_metric('delete_rps_max', function() return box.stat().DELETE.rps end, graphite.max)`
+``` lua
+graphite.add_sec_metric('delete_rps_max', function() return box.stat().DELETE.rps end, graphite.max)
+
+graphite.add('requests', 1)
+
+graphite.max_per_min('max_mysql_query_time', query_time)
 
-`graphite.add('requests', 1)`
+-- graphite.wrap_func adds for function dostuff following stats with one line:
```

Comment:
Please rephrase this to something like "An utility function, handy for adding the following set of metrics with a single line of code".
Why this particular set anyway?  How many custom rpm metrics do you have?